README.md

```diff
@@ -147,160 +154,237 @@ print("Training set accuracy = " + str(evaluator.evaluate(predictionAndLabels)))
-    ```
+```python
+from sparkdl import readImages, TFImageTransformer
+import sparkdl.graph.utils as tfx  # strip_and_freeze_until was moved from sparkdl.transformers to sparkdl.graph.utils in 0.2.0
+from sparkdl.transformers import utils
+import tensorflow as tf
+
+graph = tf.Graph()
+with tf.Session(graph=graph) as sess:
+    image_arr = utils.imageInputPlaceholder()
+    resized_images = tf.image.resize_images(image_arr, (299, 299))
+    # the following step is not necessary for this graph, but can be for graphs with variables, etc
+    frozen_graph = tfx.strip_and_freeze_until([resized_images], graph, sess, return_graph=True)
+
+transformer = TFImageTransformer(inputCol="image", outputCol="predictions", graph=frozen_graph,
+                                 inputTensor=image_arr, outputTensor=resized_images,
+                                 outputMode="image")
+
+image_df = readImages(sample_img_dir)
+processed_image_df = transformer.transform(image_df)
+```
 
-3.  For Keras users
+##### For Keras users
+For applying Keras models in a distributed manner using Spark, [`KerasImageFileTransformer`](link_here) works on TensorFlow-backed Keras models. It 
+* Internally creates a DataFrame containing a column of images by applying the user-specified image loading and processing function to the input DataFrame containing a column of image URIs
+* Loads a Keras model from the given model file path 
+* Applies the model to the image DataFrame
 
-    #### Images
+The difference in the API from `TFImageTransformer` above stems from the fact that usual Keras workflows have very specific ways to load and resize images that are not part of the TensorFlow Graph.
 
-    For applying Keras models to images in a distributed manner using Spark,
-    `KerasImageFileTransformer` works on TensorFlow-backed Keras models. It
+To use the transformer, we first need to have a Keras model stored as a file. For this notebook we'll just save the Keras built-in InceptionV3 model instead of training one.
 
-    1.  Internally creates a DataFrame containing a column of images by applying the user-specified image loading and processing function to the input DataFrame containing a column of image URIs
-    2.  Loads a Keras model from the given model file path
-    3.  Applies the model to the image DataFrame
 
-    The difference in the API from `TFImageTransformer` above stems from the fact that usual Keras workflows have very specific ways to load and resize images that are not part of the TensorFlow Graph.
+```python
+from keras.applications import InceptionV3
 
-    To use the transformer, we first need to have a Keras model stored as a file. For this example we'll just save the Keras built-in InceptionV3 model instead of training one.
+model = InceptionV3(weights="imagenet")
+model.save('/tmp/model-full.h5')  # saves to the local filesystem
+# move to a permanent place for future use
+dbfs_model_path = 'dbfs:/models/model-full.h5'
```

Comment:
we don't want the dbfs stuff here because this README is for general spark usage. let's revert this cell to the previous version in this README (okay in the notebook). this is precisely the kind of the difference we have between the README and databricks notebooks.